interview-questions.md

```diff
@@ -8,44 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a Ruby class?
 
-Your answer:
+Your answer: Using super in a sub class, allows the subclass to inherit the parameters from the parent class. We do not need to add the inhereted parameters to the subclass, but when adding new parameters to the subclass, we must also initialize and add them to our getter/setter methods if we have them. 
 
-Researched answer:
+Researched answer: Super can only be called inside a method. A superclass can have many subclasses. We can also use super() if the parent class doesn't have the same number of parameters.
 
 2. What is a gem?
 
-Your answer:
+Your answer: A ruby library of code snippets. There are many many different gems that allow us to easily use the codes provided. 
 
-Researched answer:
+Researched answer: Ruby gems are basically software that we can install and use. They provide us premade custom methods to make life easier. Gems are open source, and anyone can create them. 
 
 3. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: A relational database is a database system that stores data relative to its other data. For example PostgreSQL.
 
-Researched answer:
+Researched answer: In relational databases, data is stored in tabs and columns. In non-relational databases, data is stored in individual files that are unrelated to eachother. 
 
 4. What are primary keys? Why purpose does a primary key serve?
 
-Your answer:
+Your answer: Primary keys the ID keys of data in a database.
```

Comment:
I would elaborate some more on what purpose they serve and why they are important